Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -173,7 +173,7 @@ uplink-antimov-law-desc = A very dangerous Lawset to use when you want to cause
 
 # Implants
 uplink-storage-implanter-name = Storage Implanter
-uplink-storage-implanter-desc = Hide goodies inside of yourself with new bluespace technology!
+uplink-storage-implanter-desc = Hide goodies inside of yourself with new bluespace technology! Recent limitations have made it impossible to store both high risk, and storage containing items inside it.
```

Comment:
probably best to just move it to the deltav namespace then change the yaml to something like `deltav-uplink-storage-implanter-desc` because ftl comments funny